task/build-paketo-ubi-builder/0.1/build-paketo-ubi-builder.yaml

```diff
@@ -0,0 +1,527 @@
+      type: string
+    - name: STORAGE_DRIVER
+      description: Storage driver to configure for buildah
+      default: vfs
+      type: string
+    - name: TLSVERIFY
+      description: Verify the TLS on the registry endpoint (for push/pull to a non-TLS registry)
+      type: string
+      default: "true"
+    - name: caTrustConfigMapKey
+      description: The name of the key in the ConfigMap that contains the CA bundle
+        data.
+      type: string
+      default: ca-bundle.crt
+    - name: caTrustConfigMapName
+      description: The name of the ConfigMap to read CA bundle data from.
+      type: string
+      default: trusted-ca
+  results:
+    - name: "IMAGE_URL"
+      description: "Image repository and tag where the built image was pushed"
+      type: string
+    - name: "IMAGE_DIGEST"
+      description: "Digest of the image just built"
+      type: string
+    - name: "IMAGE_REF"
+      description: "Image reference of the built image"
+      type: string
+    - name: "BASE_IMAGES_DIGESTS"
+      description: "Digests of the base images used for build"
+      type: string
+    - name: "SBOM_BLOB_URL"
+      description: "SBOM Image URL"
+      type: string
+  volumes:
+    - name: ssh
+      secret:
+        secretName: "multi-platform-ssh-$(context.taskRun.name)"
+    - name: shared
+      emptyDir: {}
+    - name: varlibcontainers
+      emptyDir: {}
+    - configMap:
+        items:
+          - key: $(params.caTrustConfigMapKey)
+            path: ca-bundle.crt
+        name: $(params.caTrustConfigMapName)
+        optional: true
+      name: trusted-ca
+    - name: workdir
+      emptyDir: {}
+  stepTemplate:
+    env:
+      - name: BUILDER_IMAGE
+        value: $(params.BUILDER_IMAGE)
+      - name: CONTEXT
+        value: $(params.CONTEXT)
+      - name: HERMETIC
+        value: $(params.HERMETIC)
+      - name: IMAGE
+        value: $(params.IMAGE)
+      - name: PLATFORM
+        value: $(params.PLATFORM)
+      - name: SOURCE_CODE_DIR
+        value: $(params.SOURCE_CODE_DIR)
+      - name: STORAGE_DRIVER
+        value: $(params.STORAGE_DRIVER)
+      - name: TLSVERIFY
+        value: $(params.TLSVERIFY)
+
+    volumeMounts:
+      - mountPath: /shared
+        name: shared
+      - mountPath: /var/workdir
+        name: workdir
+  steps:
+      - name: use-trusted-artifact
+        image: quay.io/redhat-appstudio/build-trusted-artifacts:latest@sha256:81c4864dae6bb11595f657be887e205262e70086a05ed16ada827fd6391926ac
+        args:
+          - use
+          - $(params.SOURCE_ARTIFACT)=/var/workdir/source
+          - $(params.CACHI2_ARTIFACT)=/var/workdir/cachi2
+      - args:
+          - "$(params.BUILD_ARGS[*])"
+        image: "quay.io/konflux-ci/buildah-task:latest"
+        name: "run-script"
+        script: |-
+          #!/usr/bin/env bash
+          set -eu
+          set -o pipefail
+          
+          echo "##########################################################################################"
```

Comment:
Such pretty prints are not in the original task, we may want to keep consistent output in logs for users

Reply:
> Such pretty prints are not in the original task, we may want to keep consistent output in logs for users

What do you mean here as I don't understand why you say that pretty prints are not part of the original task ?

Reply:
Such `##############################` lines simplify the readability of the log

Reply:
that's matter of taste, but it's inconsistent with the other tasks now

Reply:
Having such messages logged help a lot to read the pod's log 
```
echo "##########################################################################################"
echo "### Step 1 :: Configure SSH and rsync folders from tekton to the VM"
echo "##########################################################################################"
```
I don't really understand why you reject them even if the other tasks don't use them. If some problems occur, that will be the job of the buildpack team to figure out the problem and to fix it and not others builder image teams.

Reply:
because consistent output is part of good UX, and this task is part of konflux, not a standalone project.

I also don't like when tasks throw at me decorated textual blocks, so it's also matter of  taste too.

How does it work in short lines? Konflux CLI can be sometimes too short in task owerviews.

Reply:
I am the PO of the Konflux UI team. There is a possibility that in the future we will be asked to parse the output of tasks. Keeping the look of the output consistent would make our lives easier if such requirement arises.

Reply:
I removed the "####". See: 21a133ac

Reply:
I like the idea to have a consistent messages logged. In this case, konflux's team should then propose guidelines to be able to log such messages. Here is by example what we do using java: https://howtodoinjava.com/log4j2/useful-conversion-pattern-examples/